hugo/static/.htaccess

```diff
@@ -38,9 +38,13 @@ RewriteRule ^(.*)$ https://%{SERVER_NAME}/anthology/$1 [R=301,L]
 # that will also redirect X/Z19/P17-1069.pdf -> /anthology-files/pdf/P/P17/1-69.pdf
 RewriteRule ^[A-Za-z]/[A-Za-z][0-9][0-9]/([A-Za-z])([0-9][0-9])\-([0-9][0-9][0-9][0-9])(.pdf)?$ https://www.aclweb.org/anthology/$1$2-$3 [R=301,L]
 
-# Redirect URLS with an explicit ".pdf" to the bare canonical version
+# Redirect URLs with an explicit ".pdf" to the bare canonical version
 RewriteRule ^([A-Za-z])([0-9][0-9])\-([0-9][0-9][0-9][0-9]).pdf$ https://www.aclweb.org/anthology/$1$2-$3 [R=301,L]
 
+# Redirect nested paper pages to the / version (e.g., papers/P/P19/P19-1001/ -> P19-1001/)
+# This way there is only one page. Should maintain for backward compatibility for some time after August 2019.
+RewriteRule ^papers/[A-Za-z]/[A-Za-z][0-9][0-9]/([A-Za-z])([0-9][0-9])\-([0-9][0-9][0-9][0-9])/?$ https://www.aclweb.org/anthology/$1$2-$3/ [R=301,L]
+
 #
```

Comment:
This rule redirects (with a 301) the nested links to the canonical URL with a `/` (so we get the paper page).

Reply:
This was merged in 16cd5fe2fdf0c99477adce1a1106f9020e6b57b4.